src/bin/config/default_ctants.rs

```diff
@@ -0,0 +1,5 @@
+/// Default port that Rusk GRPC-server will listen to.
+pub(crate) const PORT: &'static str = "8585";
+
+/// Default host_address that Rusk GRPC-server will listen to.
+pub(crate) const HOST_ADDRESS: &'static str = "127.0.0.1";
```

Comment:
No need to have such constants in a separate module, having in the binary (main) is good enough.

Reply:
Moved to main.rs